examples/99-advanced/customization-trame-toolbar.py

```diff
@@ -0,0 +1,155 @@
+"""
+.. _customize_trame_toolbar_example:
+
+Customize Trame toolbar
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Bring more of the power of trame to the jupyter view.
+"""
+import asyncio
+
+from trame.widgets import vuetify
+import vtk
+
+import pyvista as pv
+from pyvista.trame.ui.vuetify2 import button, select, slider, text_field
+
+###############################################################################
+# Let's first create the menu items we want to add to the trame's toolbar.
+# Here we want a "play" button that will be later connected to a slider
+# through the ``btn_play`` function. The slider itself will represent the
+# "resolution" of the model we will render, a text field where the value of
+# the "resolution" will be displayed.
+# We will also add a dropdown menu to toggle the visibility of the model.
+# The dividers are the same already used to divide and organize the toolbar.
+
+
+def custom_tools():
+    vuetify.VDivider(vertical=True, classes='mx-1')
+    button(
+        click=btn_play,
+        icon='mdi-play',
+        tooltip='Play',
+    )
+
+    slider(
+        model=("resolution", 10),
+        tooltip="Resolution slider",
+        min=3,
+        max=20,
+        step=1,
+        dense=True,
+        hide_details=True,
+        style="width: 300px",
+        classes='my-0 py-0 ml-1 mr-1',
+    )
+    text_field(
+        model=("resolution", 10),
+        tooltip="Resolution value",
+        readonly=True,
+        type="number",
+        dense=True,
+        hide_details=True,
+        style="min-width: 40px; width: 60px",
+        classes='my-0 py-0 ml-1 mr-1',
+    )
+
+    vuetify.VDivider(vertical=True, classes='mx-1')
+    select(
+        model=("visibility", "Show"),
+        tooltip="Toggle visibility",
+        items=['Visibility', ["Hide", "Show"]],
+        hide_details=True,
+        dense=True,
+    )
+
+
+###############################################################################
+# The button callback function ``btn_play`` needs to be created before starting
+# the server. This function will toggle the boolean state variable ``play``
+# and flush the server, i.e. "force" the server to see the change.
+# We will see more on the state variables in a bit, but we need to create the
+# function here otherwise the server will complain ``btn_play`` does not exist.
```

Comment:
```suggestion
# The button callback function ``button_play`` needs to be created before starting
# the server. This function will toggle the boolean state variable ``play``
# and flush the server, i.e. "force" the server to see the change.
# We will see more on the state variables in a bit, but we need to create the
# function here otherwise the server will complain ``button_play`` does not exist.
```